repasando para este sabado y modificando

Kathion-Vega opened the pull request:
la indicacion era no usar porcentajes, verdad??? Ya no me acuerdo, saludos

Commit:
repasando para este sabado y modificando

la indicacion era no usar porcentajes, verdad??? Ya no me acuerdo, saludos

RodolfoSaldivar commented on app/css/Kathia-style.css:
```diff
@@ -1,44 +1,101 @@
+  grid-template-rows: auto auto;
+  margin: 30px;
+  padding: 10px;
+  /*width: auto;
```
Si no se necesita va fuera

RodolfoSaldivar commented on app/css/Kathia-style.css:
```diff
@@ -1,44 +1,101 @@
+  grid-template-columns: 3fr 1fr;
+  grid-gap: 20px;
+  background-color: grey;
+  
```
lineas vacias se quitan

RodolfoSaldivar commented on app/css/Kathia-style.css:
```diff
@@ -1,44 +1,101 @@
+}
+
+.item3 {
+  
```
en todo el documento quitar ls lineas vacias

RodolfoSaldivar commented on app/html/Kathia-grid1.html:
```diff
@@ -0,0 +1,42 @@
+					<p>Kathia</p>
+				</div>
+			</div>
+        	<div class="grid-container right">
```
tabular

RodolfoSaldivar commented on app/html/Kathia-grid1.html:
```diff
@@ -0,0 +1,42 @@
+
+
+</body>
+</html>
```
Si se pueden usar porcentajes

Commit:
ejerccio 2 de grid

No se si tenia que ser responsivo, no.. verdad???
Ya empecera a revisar las otras tareas, gracias.

RodolfoSaldivar commented on app/css/Kathia-grid1-style.css:
```diff
@@ -0,0 +1,101 @@
+  margin: 30px;
+  padding: 10px;
+  /*width: auto;
+  height: auto;*/
```
borrar comentarios

RodolfoSaldivar commented on app/css/Kathia-grid1-style.css:
```diff
@@ -0,0 +1,101 @@
+  grid-template-columns: 3fr 1fr;
+  grid-gap: 20px;
+  background-color: grey;
+  
```
borrar lineas vacias

RodolfoSaldivar commented on app/css/Kathia-grid1-style.css:
```diff
@@ -0,0 +1,101 @@
+  border-radius: 8px;
+}
+.item1 {
+  
```
lineas

RodolfoSaldivar commented on app/html/Kathia-grid1.html:
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
```
este documentos solo dice kathia varias veces, es lo que buscabas?

RodolfoSaldivar commented on app/html/Kathia-grid2.html:
```diff
@@ -0,0 +1,163 @@
+							</div>
+						</div>
+					</div>
+					<div class="item item5">
```
tiene que tomar todo el espacio que sobra, y no es responsivo este pedazo
